lib/nerves_livebook/application.ex

```diff
@@ -26,7 +26,20 @@ defmodule NervesLivebook.Application do
 
     # Best effort create everything
     _ = File.mkdir_p(destination_dir)
-    Enum.each(["welcome.livemd", "samples"], &symlink(source_dir, destination_dir, &1))
+
+    Enum.each(
```

Comment:
There's a tradeoff with removing the samples directory and that's if any of these directories changes names or gets deleted, then the user will have broken symlinks to the old ones after they update.

I think we're stuck with one directory and one livebook in the root directory. I don't mind if we change the names, though.

Reply:
Oh got it! Yeah I agree that one directory is the best course of action then. Updates a really cool feature.